src/main/java/com/google/sps/data/RecommendationEngine.java

```diff
@@ -69,17 +63,14 @@ public static String getBestShop(List<String> listStoreIds, String inputBeverage
         }
       }
     }
-    System.out.println(
-        "4"); // ---------------------------------------------------> REMOVE AFTER FINISHING TEST
+
     if (!highestScoredStore.equals("")) {
-      System.out.println(
-          "5"); // ---------------------------------------------------> REMOVE AFTER FINISHING TEST
       return highestScoredStore;
     }
     return "No Results";
   }
 
-  public static int getDistanceMins() { // todo: get distance of store in mins
+  public static int getWalkingDistanceMins() { // todo: get distance of store in mins
```

Comment:
put a todo to return Duration instead of int here then you can also change the name to not include minutes